components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import {View, Text, Pressable} from 'react-native';
+import {Colors} from '../../utils/colors';
+
+import style from './style';
+import globalStyle from '../../assets/styles/globalStyle';
+
+const RadioButton: React.FC<{
```

Comment:
`RadioButton`, singular. Seems to accept an array called `data` hence it will render multiple buttons not just 1. I suggest fixing the name, suggestions: `RadioGroup` `RadioButtons` or similar.